regression/pages/lms/instructor_dashboard.py

```diff
@@ -0,0 +1,37 @@
+"""
+Instructor dashboard page.
+"""
+from edxapp_acceptance.pages.lms.instructor_dashboard import (
+    InstructorDashboardPage
+)
+from regression.pages.lms import BASE_URL
+
+
+class InstructorDashboardPageExtended(InstructorDashboardPage):
+    """
+    This class is an extended class of Instructor Dashboard Page,
+    where we add methods that are different or not used in
```

Comment:
I think just the first line of docstring is fine. If you want, you could say something about how this has extra navigation that's important to e2e tests.